### ISSUE TITLE
Transferring the PM system to TFM.

### ISSUE THREAD

**Issue by mgejay** [TotalFreedom/TotalFreedomMod#2089]
The reasons I'd like to do this:

1. It's not very visible and formatting it to be could be a good idea.
2. We can implement it in and make it easier for admins to spy on it with a command.
3. People can't msg everybody with '*'.

**Comment by Wild1145** [comment]
So you want to migrate the Essentials /msg commands to TFM?

**Comment by mgejay** [comment]
@Wild1145 Yes.

**Comment by Wild1145** [comment]
Yeah, I am not opposed to the idea. In an ideal world TFM would remove the need for many if any other plugins. I see no reason why this sort of a system couldnt be migrated? Given its offlinemode design I'd probably say we just make it work for online players and scrap having a "MailBox"?

**Comment by mgejay** [comment]
Sounds like a good idea to me.
